### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by yksingh12** [martiliones/icon-set-creator#10]
I am getting this error while generating the icons for Android. Library working fine for iOS.

```
$ iconset create ./src/Assets/common/icon.png
 INFO  Creating default icons Android...

 ERROR  TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:371:5)
    at validateString (node:internal/validators:119:11)
    at Object.resolve (node:path:1098:7)
    at /Users/yogeshsingh/Documents/Projects/ReactNative/mobile-app/node_modules/icon-set-creator/lib/utils/android.js:57:38
    at new Promise (<anonymous>)
    at exports.createAndroidIcons (/Users/yogeshsingh/Documents/Projects/ReactNative/mobile-app/node_modules/icon-set-creator/lib/utils/android.js:56:10)
    at create (/Users/yogeshsingh/Documents/Projects/ReactNative/mobile-app/node_modules/icon-set-creator/lib/create.js:44:13)
    at Command.<anonymous> (/Users/yogeshsingh/Documents/Projects/ReactNative/mobile-app/node_modules/icon-set-creator/bin/iconset.js:65:29)
    at Command.listener [as _actionHandler] (/Users/yogeshsingh/Documents/Projects/ReactNative/mobile-app/node_modules/icon-set-creator/node_modules/commander/lib/command.js:488:17)
    at /Users/yogeshsingh/Documents/Projects/ReactNative/mobile-app/node_modules/icon-set-creator/node_modules/commander/lib/command.js:1227:65 {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

Other Details:
icon-set-creator 0.1.3
react-native 0.66.2
node v16.13.0

**Comment by DanielMarkiel** [comment]
I've got the same error using node v14.18.1.

**Comment by TheEhsanSarshar** [comment]
@DanielMarkiel 
Thank you that's solved the problem

**Comment by martiliones** [comment]
I'm sorry for waiting. I just published a new `0.1.4` version with that pull request. Going to rewrite everything in TypeScript and OOP. If you want to help me with TS or Testing, PM me in Telegram (@martiliones)

Feel free to reopen this issue or open a new one if you're having troubles with it :)
